pkg/didcomm/envelope/packager.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package envelope
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/btcsuite/btcutil/base58"
+
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/crypto"
+)
+
+// Provider contains dependencies for the base packager and is typically created by using aries.Context()
+type Provider interface {
+	Crypter() crypto.Crypter
+}
+
+// BasePackager is the basic implementation of Packager
+type BasePackager struct {
+	crypter crypto.Crypter
+}
+
+// New return new instance of wallet implementation
+func New(ctx Provider) (*BasePackager, error) {
+	crypter := ctx.Crypter()
+
+	return &BasePackager{crypter: crypter}, nil
+}
+
+// PackMessage Pack a message for one or more recipients.
+func (p *BasePackager) PackMessage(envelope *Envelope) ([]byte, error) {
+	if envelope == nil {
+		return nil, errors.New("envelope argument is nil")
+	}
+
+	var recipients [][]byte
+	for _, verKey := range envelope.ToVerKeys {
+		// TODO It is possible to have different key schemes in an interop situation
```

Comment:
Do we have an issue for this?

Reply:
this code and comment was there in the wallet originally.. they were moved to envelope as is...

I will remove this comment as different crypto's and key schemes can be implemented later.

Reply:
updated comment